### ISSUE TITLE
Add parameter for changing loss for NONE

### ISSUE THREAD

**Issue by isabelleaugenstein** [sheffieldnlp/stance-conditional#4]
...because it doesn't contribute to Macro F1

Ignore the NONE training examples: 
- multiplying the total loss by 1-target[0](so if the class is NONE the loss is 0)
- remove NONE examples from the training+dev corpus. 

Or downweigh them:
loss = (1-(target[0] \* (1 - alpha))) \* loss

So for target class NONE and alpha 0.1 we would get:
loss = (1-(1_0.9))_loss = 0.1*loss
